src/main/java/com/hermesworld/ais/galapagos/security/impl/AuditEventsListener.java

```diff
@@ -103,13 +103,13 @@ public AuditEventsListener(AuditEventRepository auditRepository) {
     }
 
     @Override
-    public CompletableFuture<Void> handleAddTopicProducers(TopicAddProducersEvent event) {
-        return FutureUtil.noop();
+    public CompletableFuture<Void> handleAddTopicProducer(TopicAddProducersEvent event) {
+        return handleTopicEvent(event, "TOPIC_PRODUCER_APPLICATION_ADDED");
```

Comment:
Should be `GalapagosAuditEventType.TOPIC_PRODUCER_APPLICATION_ADDED.name()` (no extra string constant here - is error prone)